src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritDynamicUrlProcessor.java

```diff
@@ -238,12 +237,11 @@ private static String regexEscapted(char symbol) {
      * since the last fetch, it returns null.
      *
      * @param gerritTriggerConfigUrl the URL to fetch
-     * @param serverName name of the Gerrit server.
      * @return a list of GerritProjects if successful, or null if no change
      * @throws ParseException when the fetched content couldn't be parsed
      * @throws IOException for all other kinds of fetch errors
      */
-    public static List<GerritProject> fetch(String gerritTriggerConfigUrl, String serverName)
+    public static List<GerritProject> fetch(String gerritTriggerConfigUrl)
```

Comment:
I get a strange twitch in my eye whenever a public method signature gets changed without a deprecated old signature left intact.
In this case I doubt anyone external uses it so it is most likely fine.
While you're in here could you add an `@Restricted(NoExternalUse.class)` on the class if that's possible in this version of core otherwise on each public method signature.